js/controllers/menu.js

```diff
@@ -131,6 +132,11 @@ MenuController.prototype.saveas_ = function() {
   return false;
 };
 
+MenuController.prototype.openShortcuts_ = function(e, tab) {
```

Comment:
`e, tab` looks unused. Can remove it.

Reply:
Yeap, makes sense! Removed both arguments